### ISSUE TITLE
How to draw side strokes ? 

### ISSUE THREAD

**Issue by ThuanNIT** [recharts/recharts#3980]
I want to combine the broken lines, but I haven't found a way yet.

![AreaChart](https://github.com/recharts/recharts/assets/63941443/2440f850-a707-4488-8773-121b7109c640)

It will be like this
![2](https://github.com/recharts/recharts/assets/63941443/a174891a-dbd3-48e0-b248-4254d1cf5b56)

**Comment by ThuanNIT** [comment]
I tried a way to add strokes to the style but it didn't work as I expected..

![code style](https://github.com/recharts/recharts/assets/63941443/42af843c-0bef-4fd9-80c0-7c636ded5ab4)

![set Style](https://github.com/recharts/recharts/assets/63941443/3c98ace3-dc3d-46ad-ae92-f122142164d8)

**Comment by ckifer** [comment]
I don't think this is possible - its either a stroke on the entire thing or no stroke as of right now

**Comment by ThuanNIT** [comment]
yes . I mean .  I want draw a stroke in the endpoint  . 
![image](https://github.com/recharts/recharts/assets/63941443/914b69e5-e550-46a0-b80a-22acc6598396)
